src/main/java/uk/gov/hmcts/reform/sscs/personalisation/Personalisation.java

```diff
@@ -83,6 +83,16 @@
         personalisation.put(PHONE_NUMBER, config.getHmctsPhoneNumber());
         personalisation.put(CCD_ID, StringUtils.defaultIfBlank(ccdResponse.getCcdCaseId(), StringUtils.EMPTY));
 
+        // Some templates (noteably letters) can be sent out before the SC Ref is added to the case
+        // this allows those templates to be populated with either the CCD Id or SC Ref
```

Comment:
strange comment.
What is the difference between this logic and `getAppealReference` that populates `APPEAL_REF`?